tests/track/platform_trackers/test_mlflow_tracker.py

```diff
@@ -0,0 +1,325 @@
+            kind="job",
+            image="mlrun/mlrun",
+            requirements=["mlflow"],
+        )
+
+        mlflow_run = mlflow.last_active_run()
+        imported_run = MLFlowTracker.import_run(
+            project=project,
+            pointer=mlflow_run.info.run_id,
+            function_name="import_run_test",
+        )
+
+        _validate_run(run=imported_run)
+
+
+@pytest.mark.parametrize("handler", [xgb_run, lgb_run])
+def test_import_model(rundb_mock, handler):
+    """
+    This test is for importing a model logged by mlflow into mlrun.
+    first we run some code that's logged by mlflow, then we import the logged model
+    to mlrun, and then we validate
+    """
+    mlflow.environment_variables.MLFLOW_EXPERIMENT_NAME.set(handler.__name__)
+    with tempfile.TemporaryDirectory() as test_directory:
+        mlflow.set_tracking_uri(test_directory)  # Tell mlflow where to save logged data
+
+        # Run mlflow code
+        handler()
+
+        # Create a project for this tester:
+        project = mlrun.get_or_create_project(name="default", context=test_directory)
+
+        # Create a MLRun function using the tester source file (all the functions must be located in it):
+        project.set_function(
+            func=__file__,
+            name=f"{handler.__name__}-test",
+            kind="job",
+            image="mlrun/mlrun",
+            requirements=["mlflow"],
+        )
+
+        # Access model's uri through mlflow's last run
+        mlflow_run = mlflow.last_active_run()
+        model_uri = mlflow_run.info.artifact_uri + "/model"
+
+        key = "test_model"
+        MLFlowTracker.import_model(
+            project=project,
+            pointer=model_uri,
+            key=key,
+            metrics=mlflow_run.data.metrics,
+        )
+
+        # Validate model was logged into project
+        assert project.get_artifact(key)
+
+
+@pytest.mark.parametrize("handler", [xgb_run, lgb_run, simple_run])
+def test_import_artifact(rundb_mock, handler):
+    """
+    This test is for importing an artifact logged by mlflow into mlrun.
+    first we run some code that's logged by mlflow, then we import logged artifacts
+    to mlrun, and then we use validate by comparing to original artifacts
+    """
+    mlflow.environment_variables.MLFLOW_EXPERIMENT_NAME.set(handler.__name__)
+    with tempfile.TemporaryDirectory() as test_directory:
+        mlflow.set_tracking_uri(test_directory)  # Tell mlflow where to save logged data
+
+        # Run mlflow code
+        handler()
+
+        # Create a project for this tester:
+        project = mlrun.get_or_create_project(name="default1", context=test_directory)
+
+        # Create a MLRun function using the tester source file (all the functions must be located in it):
+        project.set_function(
+            func=__file__,
+            name=f"{handler.__name__}-test",
+            kind="job",
+            image="mlrun/mlrun",
+            requirements=["mlflow"],
+        )
```

Comment:
no need to set a function in order to import an artifact to a project. same for importing a model.